### PR TITLE
fix: remove twitter handle from footer, add webshare api

### PR DESCRIPTION
**Pull request by ovbm**
- Removes Twitter from our footer 
- uses web Share API instead of our own share overlay if a mobile device (ios or android) is used

tested on love

### CODE REVIEW

File: apps/www/components/ActionBar/ShareButtons.js [hunk 1]
```diff
@@ -46,33 +46,6 @@ const ShareButtons = ({
     }
   }, [copyLinkSuffix])
```
Comment (ovbm) [inline apps/www/components/ActionBar/ShareButtons.js:48]: The only place this was ever used was the end of a series overview. I removed that so the ShareButtons component now only ever renders the sharebuttons without a native app fallback.

File: apps/www/components/ActionBar/index.js [hunk 2]
```diff
@@ -382,20 +404,9 @@ const ActionBar = ({
...
-            payload: {
-              title: document.title,
-              url: shareUrl,
-              subject: emailSubject,
-              dialogTitle: t('article/share/title'),
-            },
-          })
+        handleShareClick()
+        if (isNativeApp) {
           e.target.blur()
```
Comment (annatraussnig) [inline apps/www/components/ActionBar/index.js:409]: it kinda bothers me that the if clause in the `handleShareClick` function is the same as the one to blur the event target (`if (inNativeApp)`).
I would pass the event/event.target to `handleShareClick`

**Review by annatraussnig — APPROVED** [review]
Code looks good. Didn't test on mobile.

File: apps/www/components/ActionBar/index.js (hunk 2 above)
Comment (ovbm) [reply apps/www/components/ActionBar/index.js:409]: yeah i passed it in before, don't know why i removed it on a later refactor...

Comment (ovbm) [reply apps/www/components/ActionBar/index.js]: done

**Pull request merged**